src/pages/HomeAbout.js

```diff
@@ -17,19 +18,58 @@ class HomeAbout extends Component {
+                  }, [
+                    span({ className: 'glyphicon glyphicon-download' }),
+                    'Download DUOS Library Card Agreement']),
+                ]),
+              ]),
+                p({ style: { fontWeight: '500' } }, [
+                  'To issue your researchers this Library Card pre-authorization, please sign and send the DUOS Library Card Agreement (above) along with a list of the first and last name, and eRA Commmons ID of each researcher you will issue this privilege, to: ',
+                  h(Mailto, { email: 'DUOS-support@broadinstitute.zendesk.com' }),
+                  p({ style: { fontWeight: '400' } }, ["Please note, this agreement is non-negotiable."]),
+
+                div({ className: "home-content" }, [
+                  h3({}, ["DUOS & Library Card"]),
+                  p( { className: "home-sections-description" }, ["Bringing the two together for a data access revolution"]),
+                  p( {style: { fontWeight: '400' }}, ["The current data access request process incurs delays from and unncessary burden on Signing Officials and is further slowed by providing data access committees (DACs) with ambiguous and non-compatible terms which they are obligated to interpret and evaluate to maintain compliance."]),
+                  img({ src: "/images/DUOS_LC_Process.png", className: "col-lg-12 col-md-12 col-sm-12 col-xs-12 home-content-graphic", alt: "About icon" }),
+                  p( {style: { fontWeight: '450' }}, ["Combined, the DUOS Library Card Agreement and DUOS system significantly alleviate the burden of work on Signing Officials and DACs while enhancing their compliant function and expediting appropriate access for researchers. "]),
+                ]),
+
+                div({ className: "home-content" }, [
+                  h3({}, ["Machine Readable Consent Forms & DUOS"]),
+                  p( { className: "home-sections-description" }, ["Standardizing data sharing language for expedited and compliant sharing"]),
+                  p( {style: { fontWeight: '400' }}, ["To date, many consent forms contain ambiguous or region-specific data sharing language, making widespread data sharing and aggregation of datasets difficult or near impossible."]),
+                  span({}, []),
+                  a({ href: 'https://blog.primr.org/is-your-data-sharing-consent-language-transparent-and-machine-readable/', target: '_blank' }, 'GA4GH\'s Machine Readable Consent Form Initiative'),
+                  p( {style: { fontWeight: '400' }}, ["This initiative provides standard data sharing language for IRBs, funders, and investigators to use in their consent forms, based on the Data Use Ontology (DUO)."]),
+                  img({ src: "/images/machine readable consent forms.png", className: "col-lg-12 col-md-12 col-sm-12 col-xs-12 home-content-graphic", alt: "About icon" }),
```

Comment:
Avoid spaces in file names:
```suggestion
                  img({ src: "/images/machine_readable_consent_forms.png", className: "col-lg-12 col-md-12 col-sm-12 col-xs-12 home-content-graphic", alt: "About icon" }),
```